help/grandorgue.xml

```diff
@@ -9913,9 +9913,22 @@ on which the pipes of the are placed.
           <term>Percussive</term>
           <listitem>
             <para>
-(boolean, required) If false, the samples are played as is (without
-any loop/release handling)
-     </para>
+	      (boolean, required) If false, the samples are played as is
+	      (without any loop/release handling).
+	    </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>HasIndependentRelease</term>
+          <listitem>
+            <para>
+	      (boolean, default false or inherited from the Organ, Windchest or
```

Comment:
It's not really inherited from Organ or Windchest, right? It's only inherited from the Rank level and down to the pipe level, which is very much fine for me, but the wording of the help text should match reality I think.

Reply:
Both `Percussive` and `HasIndependentRelease` may be also defined at the Organ or Windchest level. For some kind of instruments (ex. harpsichords) it may be useful.

Reply:
>  For some kind of instruments (ex. harpsichords) it may be useful.

Maybe. But as far as I've tested it it's not possible with this PR to write such lines to [Organ] or [Windchest999] sections, so I suggest just to change the help text like I mentioned above.

Reply:
@oleg68 Ok, so now it would be possible to actually set the Percussive= and HasIndependentRelease= in the [Organ] and [Windchest999] sections and have it inherited to rank and pipe level. But it's not limited at that.

Both the `Percussive` and `HasIndependentRelease` now needs to be added to the [Organ] and [Windchest999] help sections too since they are available to set through the odf. Also, for the Windchest help section all the PitchTuning, PitchCorrection etc that comes with the GOPipeConfig must also be documented as they suddenly now are available for usage in an odf.

Reply:
@oleg68 Just to be clear, I'm not at all opposed to this development, I just want it well documented as it's a quite big change. I've also prepared GoOdf to be able to handle all the new properties, but I'll wait with pushing any changes until this PR is merged.